pycode/memilio-epidata/memilio/epidata_test/test_epidata_get_vaccination_data.py

```diff
@@ -171,7 +172,7 @@ def test_sanity_checks(self, mockv):
                 "LandkreisId_Impfort": ['05754', '1', '2', '3', '4'],
                 "Altersgruppe": ["01-59", "01-59", "01-59", "01-59", "01-59"],
                 "Impfschutz": [1, 1, 2, 3, 1],
-                "Anzahl": [10000, 1, 2, 3, 4]})
+                "Anzahl": [10000., 1, 2, 3, 4]})
```

Comment:
```suggestion
                "Anzahl": [10000., 1., 2., 3., 4.]})
```
Is there a reason why you changed this only for one value and not the others?